pkg/apis/appmesh/v1beta1/types.go

```diff
@@ -399,7 +399,7 @@ type VirtualNodeCondition struct {
 	Reason *string `json:"reason,omitempty"`
 	// A human readable message indicating details about the transition.
 	// +optional
-	Message *string `json:"reason,omitempty"`
+	Message *string `json:"message,omitempty"`
```

Comment:
@stefanprodan AFAICT, this is the only attribute that substantively was changed, yes? All the other changes are result of go fmt, yeah?

Reply:
Yes, my editor (Goland) runs go fmt before a commit. In the future we should add a fmt check job to CI and validate that all go files are properly formatted, I suspect many aren't.

Reply:
In the AWS CNI we have a `check-format` target [in the Makefile](https://github.com/aws/amazon-vpc-cni-k8s/blob/8f40e8ebfb0a00f6a7330fe64f923b926dec9669/Makefile#L120-L134), and then just call that target in the [`.travis.yml`](https://github.com/aws/amazon-vpc-cni-k8s/blob/8f40e8ebfb0a00f6a7330fe64f923b926dec9669/.travis.yml#L26)